plugins/repoquery.py

```diff
@@ -83,8 +84,11 @@ def parse_arguments(args):
     parser = dnfpluginscore.ArgumentParser(RepoQueryCommand.aliases[0])
     parser.add_argument('key', nargs='?',
                         help=_('the key to search for'))
-    parser.add_argument('--repoid', metavar='REPO',
+    parser.add_argument('--repo', metavar='REPO', action='append',
```

Comment:
not mention repoid in docs but support it, please.  i.e.: `parser.add_argument('--repo', '--repoid', metavar='REPO', action='append'`. we cannot drop it inmediatelly.